test/powershell/Modules/Microsoft.PowerShell.LocalAccounts/Pester.Command.Cmdlets.LocalAccounts.LocalGroup.Tests.ps1

```diff
@@ -0,0 +1,853 @@
+        $result = Get-Command -Module Microsoft.PowerShell.LocalAccounts | % Name 
+
+        $result -contains "New-LocalGroup" | Should Be $true
+        $result -contains "Set-LocalGroup" | Should Be $true
+        $result -contains "Get-LocalGroup" | Should Be $true
+        $result -contains "Rename-LocalGroup" | Should Be $true
+        $result -contains "Remove-LocalGroup" | Should Be $true
+    }
+}
+
+Describe "Validate simple New-LocalGroup" -Tags "CI", "Feature" {
+    AfterEach {
+        RemoveTestGroups -basename TestGroupAddRemove
+    }
+
+    It "Creates New-LocalGroup using only name" {
+        $result = New-LocalGroup -Name TestGroupAddRemove
+
+        $result.Name | Should BeExactly TestGroupAddRemove
+        $result.ObjectClass | Should Be Group
+    }
+}
+
+Describe "Validate New-LocalGroup cmdlet" -Tags "Feature" {
+    AfterEach {
+        RemoveTestGroups -basename TestGroupAddRemove
+    }
+
+    It "Creates New-LocalGroup with name and description" {
+        $result = New-LocalGroup -Name TestGroupAddRemove -Description "Test Group New 1 Description"
+
+        $result.Name | Should BeExactly TestGroupAddRemove
+        $result.Description | Should BeExactly "Test Group New 1 Description"
+        $result.SID | Should Not BeNullOrEmpty
+        $result.ObjectClass | Should Be Group
+    }
+
+    It "Errors on New-LocalGroup with name collision" {
+        $sb = {
+            New-LocalGroup TestGroupAddRemove
+            return New-LocalGroup TestGroupAddRemove
+        }
+        VerifyFailingTest $sb "GroupExists,Microsoft.PowerShell.Commands.NewLocalGroupCommand"
+    }
+
+    It "Can use SID for group name" {
+        $sidName = "S-1-5-21-3949576937-491355012-4054854628-1053"
+        $result = New-LocalGroup -Name $sidName
+
+        $result | Should Not BeNullOrEmpty
+        $result.Name | Should BeExactly $sidName
+        $result.SID | Should Not BeExactly $sidName
+        $result.ObjectClass | Should Be Group
+        
+        RemoveTestGroups -basename $sidName
+    }
+
+    It "Errors on empty group name" {
+        $sb = { New-LocalGroup -Name "" }
+        VerifyFailingTest $sb "ParameterArgumentValidationError,Microsoft.PowerShell.Commands.NewLocalGroupCommand"
+    }
+
+    It "Errors on duplicate group name" {
+        $sb = {
+            New-LocalGroup -Name TestGroupAddRemove
+            New-LocalGroup -Name TestGroupAddRemove
+        }
+        VerifyFailingTest $sb "GroupExists,Microsoft.PowerShell.Commands.NewLocalGroupCommand"
+    }
+
+    It "Creates New-LocalGroup with name(256) and description(48) at max" {
+        $nameMax = "A"*256
+        $descMax = "D"*48
+
+        $result = New-LocalGroup -Name $nameMax -Description $descMax
+
+        $result.Name | Should BeExactly $nameMax
+        $result.Description | Should BeExactly $descMax
+        $result.SID | Should Not BeNullOrEmpty
+        $result.ObjectClass | Should Be Group
+
+        RemoveTestGroups -basename $nameMax
```

Comment:
same here, `RemoveTestGroups -basename $nameMax` should be put in the finally block.